website/docs/reference/sdks/index.md

```diff
@@ -103,6 +90,102 @@ If you see an item marked with a ❌ that you would find useful, feel free to re
+
+The list of official client-side SDKs is:
+
+- [Android SDK](/docs/generated/sdks/client-side/android-proxy.md)
+- [Flutter Proxy SDK](/docs/generated/sdks/client-side/flutter.md)
+- [iOS Proxy SDK](/docs/generated/sdks/client-side/ios-proxy.md)
+- [Javascript SDK](/docs/generated/sdks/client-side/javascript-browser.md)
+- [React Proxy SDK](/docs/generated/sdks/client-side/react.md)
+- [Svelte Proxy SDK](/docs/generated/sdks/client-side/svelte.md)
+- [Vue Proxy SDK](/docs/generated/sdks/client-side/vue.md)
+
+Additionally, the [Next.js SDK](/docs/generated/sdks/client-side/next-js.md) also supports a client-side mode.
+
+
+#### Client-side SDK compatibility tables
+
+Because client-side SDKs don't evaluate their features locally, some of their capabilities depend on what they are connected to.
+
+One clear difference between server-side SDKs and client-side SDKs, is that client-side SDKs don't support using fallback values in `isEnabled` calls. This is because the evaluation happens upstream from them.
+
+| Capability | [Android](/docs/generated/sdks/client-side/android-proxy.md) | [Flutter](/docs/generated/sdks/client-side/flutter.md) | [iOS](/docs/generated/sdks/client-side/ios-proxy.md) | [JavaScript](/docs/generated/sdks/client-side/javascript-browser.md) | [React](/docs/generated/sdks/client-side/react.md) | [Svelte](/docs/generated/sdks/client-side/svelte.md) | [Vue](/docs/generated/sdks/client-side/vue.md) |
+| --- | :-: | :-: | :-: | :-: | :-: | :-: | :-: |
+| **Category: Initialization** |  |  |  |  |  |  |  |
+| Async initialization | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ❌ |
+| Can block until synchronized | ✅ | ✅ | ✅ | ⭕ | ⭕ | ✅ | ✅ |
+| Default refresh interval | 10s | 15s | 15s | 15s | 15s | 30s | 30s |
+| Default metrics interval | 60s | 60s | 60s | 60s | 60s | 60s | 30s |
+| **Category: Custom Headers** |  |  |  |  |  |  |  |
+| static | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |
+| function | ✅ | ✅ | ⭕ | ✅ | ✅ (4.3) | ✅ | ✅ |
+| **Category: [Unleash Context](../reference/unleash-context)** |  |  |  |  |  |  |  |
+| Static fields (`environment`, `appName`) | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |
+| Defined fields | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |
+| Custom properties | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |
+| **Category: [Variants](../feature-toggle-variants.md)** |  |  |  |  |  |  |  |
+| Basic support | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |
+| [Custom stickiness (beta)](../stickiness.md#custom-stickiness-beta) | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |
+| **Category: Local backup** |  |  |  |  |  |  |  |
+| File based backup | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |
+| **Category: Usage metrics** |  |  |  |  |  |  |  |
+| Can disable metrics | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |
+| Client registration | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |
+| Basic usage metrics (yes/no) | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |
+| [Impression data](../impression-data.md) | ⭕ | ✅ | ⭕ | ⭕ | ⭕ | ⭕ | ✅ |
+| **Category: Bootstrap (beta)** |  |  |  |  |  |  |  |
+| Bootstrap from file | ✅ | ✅ | ✅ | ⭕ | ✅ | ✅ | ✅ |
```

Comment:
I thought that bootstrap is just an object. You can read this object from a file but it doesn't seem to be Unleash client responsibility to reach out to a file. I'd rather pipe the output of a file to Unleash client (Unix philosophy)